### ISSUE TITLE
iOS crash when space in project path

### ISSUE THREAD

**Issue by MangoRabbin** [flutter/flutter#27074]
When I run flutter of iOS version, it crashes with fatal error, I tried to solve for a long time but I couldn't solve. How can I fix it?]

Error Message

```
Launching lib/main.dart on iPhone 7 in debug mode...
Running pod install...                                       1.8s
CocoaPods' output:
↳
      Preparing

    Analyzing dependencies

    Inspecting targets to integrate
      Using `ARCHS` setting to build architectures of target `Pods-Runner`: (``)

    Fetching external sources
    -> Fetching podspec for `Flutter` from `.symlinks/flutter/ios`
    -> Fetching podspec for `firebase_auth` from `.symlinks/plugins/firebase_auth/ios`
    -> Fetching podspec for `firebase_core` from `.symlinks/plugins/firebase_core/ios`
    -> Fetching podspec for `path_provider` from `.symlinks/plugins/path_provider/ios`

    Resolving dependencies of `Podfile`

    Comparing resolved specification to the sandbox manifest
      A Firebase
      A FirebaseAnalytics
      A FirebaseAuth
      A FirebaseAuthInterop
      A FirebaseCore
      A FirebaseInstanceID
      A Flutter
      A GTMSessionFetcher
      A GoogleAppMeasurement
      A GoogleUtilities
      A firebase_auth
      A firebase_core
      A nanopb
      A path_provider

    Downloading dependencies

    -> Installing Firebase (5.16.0)
      > Copying Firebase from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/Firebase/5.16.0-749a8` to `Pods/Firebase`

    -> Installing FirebaseAnalytics (5.5.0)
      > Copying FirebaseAnalytics from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/FirebaseAnalytics/5.5.0-d35d4` to `Pods/FirebaseAnalytics`

    -> Installing FirebaseAuth (5.3.0)
      > Copying FirebaseAuth from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/FirebaseAuth/5.3.0-c9b3f` to `Pods/FirebaseAuth`

    -> Installing FirebaseAuthInterop (1.0.0)
      > Copying FirebaseAuthInterop from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/FirebaseAuthInterop/1.0.0-0ffa5` to `Pods/FirebaseAuthInterop`

    -> Installing FirebaseCore (5.2.0)
      > Copying FirebaseCore from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/FirebaseCore/5.2.0-ea2d1` to `Pods/FirebaseCore`

    -> Installing FirebaseInstanceID (3.4.0)
      > Copying FirebaseInstanceID from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/FirebaseInstanceID/3.4.0-97ea7` to `Pods/FirebaseInstanceID`

    -> Installing Flutter (1.0.0)

    -> Installing GTMSessionFetcher (1.2.1)
      > Copying GTMSessionFetcher from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/GTMSessionFetcher/1.2.1-32aec` to `Pods/GTMSessionFetcher`

    -> Installing GoogleAppMeasurement (5.5.0)
      > Copying GoogleAppMeasurement from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/GoogleAppMeasurement/5.5.0-621f3` to `Pods/GoogleAppMeasurement`

    -> Installing GoogleUtilities (5.3.7)
      > Copying GoogleUtilities from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/GoogleUtilities/5.3.7-111a0` to `Pods/GoogleUtilities`

    -> Installing firebase_auth (0.0.1)

    -> Installing firebase_core (0.0.1)

    -> Installing nanopb (0.3.901)
      > Copying nanopb from `/Users/rabbin/Library/Caches/CocoaPods/Pods/Release/nanopb/0.3.901-2901f` to `Pods/nanopb`

    -> Installing path_provider (0.0.1)
      - Running pre install hooks
    [!] Attempt to read non existent folder `/Users/rabbin/iCloud Drive(아카이브)/Documents/Privacy-of-Animal/ios/Pods/Firebase`.

    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/path_list.rb:52:in `read_file_system'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/path_list.rb:35:in `files'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/path_list.rb:129:in `relative_glob'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/path_list.rb:91:in `glob'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/file_accessor.rb:281:in `license'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/pod_dir_cleaner.rb:72:in `map'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/pod_dir_cleaner.rb:72:in `used_files'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/pod_dir_cleaner.rb:51:in `clean_paths'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/sandbox/pod_dir_cleaner.rb:18:in `clean!'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer/pod_source_installer.rb:156:in `clean_installation'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer/pod_source_installer.rb:72:in `clean!'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer.rb:385:in `each'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer.rb:385:in `clean_pod_sources'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer.rb:166:in `block in download_dependencies'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/user_interface.rb:64:in `section'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer.rb:162:in `download_dependencies'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/installer.rb:117:in `install!'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/command/install.rb:41:in `run'
    /Library/Ruby/Gems/2.3.0/gems/claide-1.0.2/lib/claide/command.rb:334:in `run'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/lib/cocoapods/command.rb:52:in `run'
    /Library/Ruby/Gems/2.3.0/gems/cocoapods-1.5.3/bin/pod:55:in `<top (required)>'
    /usr/local/bin/pod:22:in `load'
    /usr/local/bin/pod:22:in `<main>'
```

Flutter Doctor -v

```
✓] Flutter (Channel beta, v1.0.0, on Mac OS X 10.14.2 18C54, locale ko-KR)
    • Flutter version 1.0.0 at /Users/rabbin/development/flutter
    • Framework revision 5391447fae (8 weeks ago), 2018-11-29 19:41:26 -0800
    • Engine revision 7375a0f414
    • Dart version 2.1.0 (build 2.1.0-dev.9.4 f9ebf21297)

[!] Android toolchain - develop for Android devices (Android SDK 28.0.3)
    • Android SDK at /Users/rabbin/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] iOS toolchain - develop for iOS devices (Xcode 10.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 10.1, Build version 10B61
    • ios-deploy 2.0.0
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 31.3.3
    • Dart plugin version 182.5124
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-1248-b01)

[✓] IntelliJ IDEA Community Edition (version 2018.2.6)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 30.0.2
    • Dart plugin version 182.5124

[✓] VS Code (version 1.30.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 2.21.1

[✓] Connected device (2 available)
    • Android SDK built for x86 • emulator-5554                        • android-x86 • Android 9 (API 28) (emulator)
    • iPhone 7                  • 69D0DE63-7431-425A-9BD7-5DFC8B5512FB • ios         • iOS 12.1 (simulator)

! Doctor found issues in 1 category.
```

**Comment by zoechi** [comment]
>  `/Users/rabbin/iCloud Drive(아카이브)/Documents/Privacy-of-Animal/ios/Pods/Firebase`.

Can you please try moving the project to a folder that does not contain spaces or unicode characters?

**Comment by baeharam** [comment]
He is my friend, it was solved by ur way, thx!

**Comment by zoechi** [comment]
I reopen because this is a legit bug

Similar to #26891 (for Gradle)

**Comment by jmagman** [comment]
Probably fixed with https://github.com/CocoaPods/CocoaPods/commit/420fac3526173c52df14b59b8956f0926f1a1a72, which was fixed in CocoaPods 1.8.0.beta.1.  Not a Flutter issues.
